commit 75249653108ea6c722a31395357b4e7feb7cd7b1
Author: Leonard Lee <leonard.lee@digia.com>
Date:   Thu Dec 13 09:42:46 2012 +0100

    Elaborate shortDayName(), etc using default locale.
    
    QDate::toString() should explain QDate::shortDayName() and
    QDate::shortMonthName() will be localized name using the
    default locale from the system.
    
    Task-number: QTBUG-28522
    Change-Id: I3d475a11066b181945d85ace4d3cbb4d8fa19b73
    (cherry picked from commit
    qtbase/4cf112b641f7ea9fb1ba87cefcef7ee98b6b0831)
    Reviewed-by: Liang Qi <liang.qi@digia.com>

diff --git a/src/corelib/tools/qdatetime.cpp b/src/corelib/tools/qdatetime.cpp
index 167a6c5..65404b4 100644
--- a/src/corelib/tools/qdatetime.cpp
+++ b/src/corelib/tools/qdatetime.cpp
@@ -544,8 +544,8 @@ int QDate::weekNumber(int *yearNumber) const
     \i 12 = "Dec"
     \endlist
 
-    The month names will be localized according to the system's locale
-    settings.
+    The month names will be localized according to the system's default
+    locale settings.
 
     \sa toString(), longMonthName(), shortDayName(), longDayName()
 */
@@ -601,8 +601,8 @@ QString QDate::shortMonthName(int month)
     \i 12 = "December"
     \endlist
 
-    The month names will be localized according to the system's locale
-    settings.
+    The month names will be localized according to the system's default
+    locale settings.
 
     \sa toString(), shortMonthName(), shortDayName(), longDayName()
 */
@@ -656,8 +656,8 @@ QString QDate::longMonthName(int month)
     \i 7 = "Sun"
     \endlist
 
-    The day names will be localized according to the system's locale
-    settings.
+    The day names will be localized according to the system's default
+    locale settings.
 
     \sa toString(), shortMonthName(), longMonthName(), longDayName()
 */
@@ -711,8 +711,8 @@ QString QDate::shortDayName(int weekday)
     \i 7 = "Sunday"
     \endlist
 
-    The day names will be localized according to the system's locale
-    settings.
+    The day names will be localized according to the system's default
+    locale settings.
 
     \sa toString(), shortDayName(), shortMonthName(), longMonthName()
 */
@@ -762,8 +762,8 @@ QString QDate::longDayName(int weekday)
     If the \a format is Qt::TextDate, the string is formatted in
     the default way. QDate::shortDayName() and QDate::shortMonthName()
     are used to generate the string, so the day and month names will
-    be localized names. An example of this formatting is
-    "Sat May 20 1995".
+    be localized names using the default locale from the system. An
+    example of this formatting is "Sat May 20 1995".
 
     If the \a format is Qt::ISODate, the string format corresponds
     to the ISO 8601 extended specification for representations of
